packages/admin-rte/src/core/Rte.tsx

```diff
@@ -238,10 +240,13 @@ const Rte: React.RefForwardingComponent<any, IProps> = (props, ref) => {
         }
     }
 
+    const rootClasses: string[] = [classes.root];
+    if (disabled) rootClasses.push(classes.disabled);
```

Comment:
disabled-selector like above

Reply:
In this case, the element is a `div` which does not support a disabled attribute. 
Only form-elements support it.